releng/ide-dmg-builder/ide-dmg-builder.sh

```diff
@@ -3,20 +3,6 @@
 BUILDDIR=$(cd $(dirname $0); pwd)
 parentdir="$(dirname "$BUILDDIR")"
 
-echo "Create DMG installer..."
-echo $parentdir
-
-$BUILDDIR/create-dmg/create-dmg \
-  --volname "Espressif-IDE" \
-  --volicon "$BUILDDIR/espressif.icns" \
-  --background "$BUILDDIR/background.png" \
-  --window-pos 200 120 \
-  --window-size 800 350 \
-  --icon-size 100 \
-  --icon "Espressif-IDE.app" 140 155 \
-  --hide-extension "Espressif-IDE.app" \
-  --app-drop-link 625 155 \
-  "$BUILDDIR/Espressif-IDE.dmg" \
-  "$parentdir/com.espressif.idf.product/target/products/com.espressif.idf.product/macosx/cocoa/x86_64/Espressif-IDE.app"
-
+echo "Create DMG..."
   
```

Comment:
As we removed this piece of code, are we not losing icon-specific information and other information which is mentioned above?

Reply:
Yes the icon specific info is lost because the dmg is now created using genisoimage utility in ubuntu and it doesn't have any option for icon specifying.

Reply:
Ok, let's figure out a way to add icon information as dmg without an icon is incomplete. If required, we could comment out the test code specific to dmg creation.